src/implementations/twig/components/checkbox/checkbox-group.html.twig

```diff
@@ -14,28 +14,17 @@
     - "name" (string) Attribute name, eg. 'data-test'
     - "value" (optional) (string) Attribute value, eg: 'data-test-1'
   Blocks:
-  - "label"
-  - "helper_text"
   - "invalid_text"
```

Comment:
well, this is a kind of unique situation, i guess that because of a design request, when we have a single checkbox we render the form error under the field, i don't think this makes sense, but i kept it as it was, with the checkbox group taking over the rendering of the form error block.
Now, what i will do is to pass the invalid_text and _invalid_icon to the checkbox-group template from the form group one but it feels they should still be listed in the checkbox-group, you could potentially decide to include this directly in certain situations, even without the form-group.